### PR TITLE
Fixed translation - see #247

### PR DESCRIPTION
**Pull request by klein0r**
see #247

### CODE REVIEW

File: admin/index_m.html [hunk 1]
```diff
@@ -101,7 +101,7 @@
                     return;
                 }
 
-                $('#dialog-bridges .modal-content').html(_('Please wait, searching for bridge.<br>' +
+                $('#dialog-bridges .modal-content').html(_('Please wait, searching for bridge.' + '<br>' +
```
Comment (foxriver76) [inline admin/index_m.html]:
```suggestion
$('#dialog-bridges .modal-content').html(_('Please wait, searching for bridge.' ) + '<br>' +
                    '<div class="progress">' +
                    '   <div class="indeterminate"></div>' +
                    '</div>');
```

should be correct. Or is it tested?

**Review by foxriver76 — CHANGES_REQUESTED** [review]

File: admin/index_m.html (hunk 1 above)
Comment (klein0r) [reply admin/index_m.html]: You are right, it was just a quick change in the browser (untested). Thanks for the review 👍

Comment (klein0r) [reply admin/index_m.html]: Please check the closing brackets, too. This was the issue.

**Comment by foxriver76** [comment]
Sorry, think you have to do it manually, it has replaced one line with three lines. 👎

**Pull request closed without merging**